treeherder/etl/job_loader.py

```diff
@@ -145,6 +148,19 @@ def transform(self, pulse_job):
             platform_src = pulse_job[v] if v in pulse_job else default_platform
             x["job"][k] = self._get_platform(platform_src)
 
+        # add some taskcluster metadata if it's available
+        # currently taskcluster doesn't pass the taskId directly, so we'll
+        # to derive it from the guid
+        try:
+            (decoded_task_id, retry_id) = job_guid.split('/')
+            real_task_id = slugid.encode(uuid.UUID(decoded_task_id))
```

Comment:
Yup, will do. This will be less confusing when we change taskcluster-treeherder to send job id information in the correct format. I'll file a followup about that.